### ISSUE TITLE
Update FFM data to version 1.2.1

### ISSUE THREAD

**Issue by kklausmeyer** [tnc-ca-geo/rivers-stream#69]
Hi Falk-  we have yet another new version of the FFM observed data to update the FFM database.  

It is available on Box [here](https://tnc.box.com/s/mqs7hprerjp0evi7nizulg5izrf33fa5) and is named "ffm_observed_20210326.csv" because it was calculated on 3/26/2021.

As with previous updates, please update the current version to the ffm database (v1.2.0) by deleting all of the data where:
`source == 'observed'`.  Then append these new data into the database.  The schema and field names should all be the same.  After the update, the new version will be v1.2.1.

After the update, please update the symbology of the river later to show which stream segments have FFM observed data.  

Let me know if you have any questions!

**Comment by postfalk** [comment]
Did the status of some stream segments changed in regard of whether they have observed data or not. Or in other words, do we need to remake the symbology as well?

**Comment by kklausmeyer** [comment]
Yes, I believe the status of several of the stream segments will have changes, so a symbology update is needed as well.  Thank you!
